File: iniciante/javascript/1160.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')


function calcularTempoDeCrescimentoPopulacional(a, b, c, d){
    let popA = a
    let popB = b
    let taxaA = c
    let taxaB = d
    let countAnos = 0

    while((popA <= popB)){
        popA += parseInt((popA * taxaA)/100)
        popB += parseInt((popB * taxaB)/100) 
        countAnos++
        if(countAnos > 100){
            return `Mais de 1 seculo.`
        }
        
    }
        return `${countAnos} anos.`
}


let testes = parseInt(lines.shift())

do{
    let entradas = lines.shift().split(' ')
    let popA = parseInt(entradas[0])
    let popB = parseInt(entradas[1])
    let taxaA = parseFloat(entradas[2])
    let taxaB = parseFloat(entradas[3])

    console.log(calcularTempoDeCrescimentoPopulacional(popA, popB, taxaA, taxaB))

}while(--testes > 0)

File: iniciante/python/1160.py
# -*- coding: utf-8 -*-

count_entradas = 0
pop_a = 0
pop_b = 0
taxa_a = 0
taxa_b = 0
count_anos = 0

qtd_entradas = int(input())
while count_entradas < qtd_entradas:
    entrada = input().split(' ')
    pop_a = int(entrada[0])
    pop_b = int(entrada[1])
    taxa_a = float(entrada[2])
    taxa_b = float(entrada[3])
    while pop_a <= pop_b:
        pop_a += (taxa_a/100 * pop_a)
        cast_a = str(pop_a).split('.')
        pop_a = int(cast_a[0])
        pop_b += (taxa_b/100 * pop_b)
        cast_b = str(pop_b).split('.')
        pop_b = int(cast_b[0])
        count_anos += 1
    if count_anos > 100:
        print('Mais de 1 seculo.')
    else:
        print(f'{count_anos} anos.')

    count_anos = 0
    count_entradas += 1
